### ISSUE TITLE
Convert RST on the fly?

### ISSUE THREAD

**Issue by rowanc1** [curvenote/curvenotejs#167]
We should be able to read RST I think.

In trying this repo out the readme was RST, but all capabilities we have:
https://github.com/simpeg-research/oldenburg-2018-AEM

A simple conversion using `rst2myst` gets us most of the way there.
https://github.com/executablebooks/rst-to-myst

**Comment by stevejpurves** [comment]
should this be a `launchpad` only conversion, or a cli thing *if* that dependency is available? (otherwise warn user and suggest installing that dep?)

**Comment by rowanc1** [comment]
Yeah, I like warning and asking if it wants to be done in a non-destructive way.
